client/src/utils/ReviewStar.jsx

```diff
@@ -24,16 +24,22 @@ function ReviewStar({ currentReviewsMetadata }) {
   let rating;
```

Comment:
Alternatively, we could just set it to 0 at the start, instead of having an else block